content/rule_book/quiz_process.md

```diff
@@ -97,7 +97,7 @@ If chair #2 erred on the toss-up, chair #2 of the team that did not err on the t
 ## Time Limits
 
 - Quizzes will start at the scheduled or announced time. Any team that is late does not receive the starting 20 points awarded for being on time.
-    - If the late team is currently quizzing in another location which is running over the time limit, they are not required to forfeit the quiz, or lose 20 points.
+    - If the late team is currently quizzing in another location which is running over the time limit, they are not required to forfeit the quiz or lose 20 points.
```

Comment:
I don't think the current wording is a problem, but "not required to" is super weird. Like, a team has the option to forfeit? lol

Reply:
This is one I think should be discussed at some point to clarify the meaning. They don't have a choice, but it could be forced I suppose, though I have a hard time coming up with a good reason that would happen outside of an emergency. If it is there, then the qualifications for forfeiture need to be outlined... e.g. X amount of minutes late outside of being in a quiz

Reply:
that IS defined, next bullet "Any team more than 10 minutes late (after the scheduled or announced time for the quiz) forfeits the quiz."

Reply:
Thank you. I completely missed that. No clue how I both forgot and missed it in my quick lookup

Reply:
I think it's clear enough.

Quiz starts at scheduled time (unless missing team(s) are/might be coming from a room in the previous time slot (which using CBQZ would make clear!!).

Between (a) the scheduled start time, (b) 10 min after, if the team shows up, they can quiz, but do not get their 20 point team bonus. (+ other specifics related to a single quizzer being late).

Once 10 min hit, that team "forfeits", which really just means they cannot quiz. They still get 1 point for 3rd. 

What happens if the 2 teams quiz and one scores < 0? They'd still get 2nd right, and 5 points?

Reply:
To me a forfeiture essentially turns it into a 2-team quiz (though without the structure change) so that means no matter how low they scored they would still receive 2nd place, and the team that forfeited would receive a 0 or would they get an automatic 3rd place and get 1 pt...

Reply:
Yeah, one thing that I think is more of a best practice than a RULE, is how QMs should handle "3 team quizzes, but scenarios within that quiz when only 2 teams can jump"

The forfeiture is one, but I think can be ignored since forfeitures are never necessary.

BUT, it's not uncommon to have a 2-person team, and both quiz out. And so late in the quiz you might have a "toss-up for 2 teams, but it's only possible for 1 team to jump".  In that scenario I don't the QM should ever provide explanatory language. Just like I don't think the QM can remind a team that it's a "jumping team bonus."

Reply:
What's the difference between a QM saying, "Question 18A is a toss-up for Team A and Team B," versus "Question 18B is a team jumping bonus for Team A"? I agree the QM shouldn't be required to explain that only 1 team can jump on a toss-up because the other team has quizzed-out. But I don't see the harm if a QM decided to do that.

I can see the argument that mentioning that the bonus is a team jumping bonus can be a good idea. PNW uses team jumping bonus, but IBQ doesn't. In an inter-district meet, adding "team jumping" is just a nice reminder. I'd never want to require it, but I don't think it's bad practice if it happens.

Moreover, is the QM be required to say: "Question 18A is a toss-up for Team A and Team B"? If yes, why? If Team C just errored, then why mention 18A is a toss-up?

Reply:
That's a really good point. The rule book doesn't "allow" a QM to say "or Team A and Team B" which pretty much every QM does.

I guess here's where I draw the distinction, "Question Number 17 is a bonus question for PNW 1" is just stating what the question is. Number, and scenario (nothing special (by omission), toss-up, bonus)

Now, I do say "Question Number 17 is a TEAM bonus for PNW 1", and "TEAM bonus" technically isn't a possible scenario.

What I don't want to be doing, is either explaining the rulebook or giving advice, pre-question, by the wording that I'm using. "Question number 17 is a team bonus for PNW1, no other teams are eligible to jump" or "you can wait for the whole question"

Reply:
I don't think the QM should be required to say "for Teams A/B/C". Required is too much. 
Prevented? I don't think that's necessary.

BUT, I don't think QMs shouldn't be "providing situational knowledge reminders"

Reply:
I see it as a way to help reduce fouls by quizzers who aren't paying attention. I don't want to require or prevent QMs from doing it. At IBQ, there would be less than zero point. In Con B at district meet 1, much value can be gained, I would think.

Reply:
That said, I think full-on preventing the QM from saying stuff like "you can wait for the whole question" is a very good thing in all contexts.

Reply:
Yeah, there are just a lot of pet things I do. 

If a fouled out quizzer is back eligible, I NEVER say "quizzer X is eligible again" but may reference them generally "all three teams are jumping".  Basically, it's on you to pay attention.

Similarly, I try to never make comments like "that was the 8th Interrogative in a row! Boy, no Quote questions yet."

Reply:
Agreed. I think saying enough so that rookies in the audience and/or parents can follow along is a good thing. Rule reminders if done consistently can be OK, depending on the competitive level context. Anything that suggests strategy decisions, always bad.